tsconfig.json

```diff
@@ -45,5 +46,8 @@
 		"./tests/unit/all.ts",
 		"./tests/unit/basicCommonJsLoading.ts",
 		"./tests/unit/require.ts"
-	]
+	],
+	"atom": {
```

Comment:
Well, it should be set to `false` then.  The Gruntfile.js rewrites the config.  We have had issues where `atom-typescript` has mucked it up.  Or other choice, don't use `atom-typescript` :expressionless: